mtools/mloginfo/sections/clients_section.py

```diff
@@ -0,0 +1,97 @@
+import re
+from collections import defaultdict
+
+from .base_section import BaseSection
+
+try:
+    from mtools.util.profile_collection import ProfileCollection
+except ImportError:
+    ProfileCollection = None
+
+
+class ClientSection(BaseSection):
+    """
+    ClientSection class.
+
+    This section goes through the logfile and extracts driver information from
+    connected clients.
+    """
+
+    name = "client driver info"
+
+    def __init__(self, mloginfo):
+        BaseSection.__init__(self, mloginfo)
+
+        helptext = 'outputs client driver information'
+        self.mloginfo.argparser_sectiongroup.add_argument('--clients',
+                                                          action='store_true',
+                                                          help=helptext)
+
+    @property
+    def active(self):
+        """Return boolean if this section is active."""
+        return(self.mloginfo.args['clients'])
+
+    def run(self):
+        """Run this section and print out information."""
+        if ProfileCollection and isinstance(self.mloginfo.logfile,
+                                            ProfileCollection):
+            print("\n    not available for system.profile collections\n")
+            return
+
+        driver_info = {}
+
+        for logevent in self.mloginfo.logfile:
+            line = logevent.line_str
+
+
+
+            pos = line.find('client metadata')
+            if pos != -1:
+                tokens = line[pos:pos + 130].split(' ')
```

Comment:
Please add a comment with context for the 130 magic number. Is that the maximum size of client metadata?

Reply:
MongoDB Internal Driver was pushing version number outside index, increased to 120 to accommodate (changed from 130)